specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json

```diff
@@ -6432,6 +6443,7 @@
       }
     },
     "SqlPoolUsage": {
+      "type": "object",
```

Comment:
Can you please share why you had to add type object?

Reply:
lint error is showing missing type:object